sagemaker-keyspaces/README.md

```diff
@@ -0,0 +1,51 @@
+# Train Machine Learning Models using Amazon Keyspaces as a Data Source.  
+
+In this lab we will provide step-by-step instruction to use SageMaker to ingest customer data from Amazon Keyspaces and train a clustering model that allowed you to segment customers. You could use this information for targeted marketing, greatly improving your business KPI.
+
+1. First, we install Sigv4 driver to connect to Amazon Keyspaces 
+
+> The Amazon Keyspaces SigV4 authentication plugin for Cassandra client drivers enables you to authenticate calls to Amazon Keyspaces ***using IAM access keys instead of user name and password***. To learn more about how the Amazon Keyspaces SigV4 plugin enables [IAM users, roles, and federated identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) to authenticate in Amazon Keyspaces API requests, see [AWS Signature Version 4 process (SigV4)](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html)
+
+2. Next, we establish a connection to Amazon Keyspaces 
```

Comment:
Style: remove all "First/Next/Finally, we..." - ie `Next, we will download retail data about customers.` -> `Download retail data about customers.`